src/ui/toolbar/ReadyIndicator.qml

```diff
@@ -0,0 +1,58 @@
+/****************************************************************************
+ *
+ * (c) 2009-2020 QGROUNDCONTROL PROJECT <http://www.qgroundcontrol.org>
+ *
+ * QGroundControl is licensed according to the terms in the file
+ * COPYING.md in the root of the source code directory.
+ *
+ ****************************************************************************/
+
+import QtQuick          2.3
+import QtQuick.Controls 1.2
+import QtQuick.Layouts  1.2
+
+import QGroundControl                       1.0
+import QGroundControl.Controls              1.0
+import QGroundControl.MultiVehicleManager   1.0
+import QGroundControl.ScreenTools           1.0
+import QGroundControl.Palette               1.0
+
+//-------------------------------------------------------------------------
+//-- Ready Indicator
+QGCComboBox {
+    function getReadyStateText() {
+        //Mirror the MAV_STATE enumeration
+        switch(_readyState) {
+        case 0:
+        case 1:
+        case 2:
+            return "Loading...";
```

Comment:
"loading" might not be really representative here, the only info provided by the system is ready / not ready for takeoff (e.g not ready != loading in case of calibration required)

Maybe we should just stick to ready / not ready (then the mavlink status msg actually provide the reason if necessary)

Reply:
Makes sense, i'll update.